source/language/classical.rst

```diff
@@ -122,6 +123,8 @@ OpenQASM evaluates expressions from left to right.
       +----------------------------------------+------------------------------------+
       | Operator                               | Operator Types                     |
       +----------------------------------------+------------------------------------+
+      | ``**``                                 | Power                              |
```

Comment:
Should this be switched? `expressionTerminator` should be the highest precedence, then power.